components/eam/src/physics/crm/crm_physics.F90

```diff
@@ -144,6 +145,8 @@ subroutine crm_physics_register()
                     longname='VT_T', readiv=.false., mixtype='dry',cam_outfld=.false.)
       call cnst_add('VT_Q', real(0,r8), real(0,r8), real(0,r8), idx_vt_q, &
                     longname='VT_Q', readiv=.false., mixtype='dry',cam_outfld=.false.)
+      call cnst_add('VT_U', real(0,r8), real(0,r8), real(0,r8), idx_vt_u, &
+                    longname='VT_U', readiv=.false., mixtype='dry',cam_outfld=.false.)
```

Comment:
for all VT_* output variables, would you consider adding more information in 'longname' attribute instead of just repeating the variable name.

Reply:
yea, good call, I'll add that.

Reply:
done.